src/hints/allocSegment.ts

```diff
@@ -0,0 +1,28 @@
+import { z } from 'zod';
+
+import { VirtualMachine } from 'vm/virtualMachine';
+import { cellRef, CellRef } from 'hints/hintParamsSchema';
+import { HintName } from 'hints/hintName';
+
+/** Zod object to parse AllocSegment hint */
+export const allocSegmentZod = z
```

Comment:
```suggestion
export const allocSegmentParser = z
```

suggesting to remove the `Zod` name as we may change one day the parser lib, I prefer to keep the naming explicit about the function that the lib

Reply:
Indeed, will make the changes in the implemented hints (AllocSegment & TestLessThan)